src/replication.thrift

```diff
@@ -478,6 +478,28 @@ struct query_disk_info_response
     4:list<disk_info> disk_infos;
 }
 
+// This request is sent from client to replica_server.
+struct replica_disk_migrate_request
+{
+    1:dsn.gpid pid
+    2:string origin_disk;
+    3:string target_disk;
+}
+
+// This response is from replica_server to client.
+struct replica_disk_migrate_response
+{
+   // Possible error:
+   // -ERR_OK: start do replica disk migrate
+   // -ERR_BUSY: current replica migration is running
+   // -ERR_INVALID_STATE: current replica partition status isn't secondary
+   // -ERR_INVALID_PARAMETERS: origin disk is equal with target disk
+   // -ERR_OBJECT_NOT_FOUND: origin or target disk isn't existed, origin disk doesn't exist current replica
```

Comment:
what if the replica not exists?

Reply:
`ERR_OBJECT_NOT_FOUND`, I will add the comment